WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListActivity.kt

```diff
@@ -5,13 +5,18 @@ import android.os.Bundle
 import android.view.MenuItem
 import kotlinx.android.synthetic.main.toolbar_main.*
 import org.wordpress.android.R
+import org.wordpress.android.ui.ActivityLauncher
 import org.wordpress.android.ui.LocaleAwareActivity
 import org.wordpress.android.ui.RequestCodes
 import org.wordpress.android.ui.posts.BasicFragmentDialog
+import org.wordpress.android.util.BackupFeatureConfig
 import org.wordpress.android.viewmodel.activitylog.ACTIVITY_LOG_REWIND_ID_KEY
+import javax.inject.Inject
 
-class ActivityLogListActivity : LocaleAwareActivity(), BasicFragmentDialog.BasicDialogPositiveClickInterface,
+class ActivityLogListActivity : LocaleAwareActivity(),
+        BasicFragmentDialog.BasicDialogPositiveClickInterface,
         BasicFragmentDialog.BasicDialogNegativeClickInterface {
+    @Inject lateinit var backupFeatureConfig: BackupFeatureConfig
```

Comment:
Question (❓ ): Is this injection required here? Did you maybe do is as a preparation for when it will be actually needed afterwards?

Reply:
Nope, not needed. I was going to do something there and decided against it. I wonder why it didn't get kicked out when I ran checkstyle. 🤷  Thanks

Reply:
Yeap this is indeed tricky, but mainly because of the `@Inject` annotation, plus the `lateinit` thingy, which makes the compiler and tooling (`IDE` in this case) think that this is used anyway. Only when you remove both and make it nullable you get to see it is unused.

Also, `CheckStyle` will not do any good here since it only works for Java files and our current `KtLint` static analysis tool is not helping as well, being old and mainly formatting related.

`Detekt` to the rescue... soon I hope! 🎉 

Unfortunately, until then, we are as good as the eye can see, authors and reviewer! 💦

Reply:
> `Detekt` to the rescue... soon I hope! 🎉

👍